packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -5,6 +5,9 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { FileProperties } from '@salesforce/source-deploy-retrieve';
+import { MetadataApiDeployStatus} from '@salesforce/source-deploy-retrieve/lib/src/client/types';
+import { LazyCollection } from '@salesforce/source-deploy-retrieve/lib/src/collections';
+import { SourceComponent } from '@salesforce/source-deploy-retrieve/lib/src/resolve';
```

Comment:
Probably don't need to import `LazyCollection` directly, that's technically an internal export. As for the other two let's import them from the top level since that follows proper versioning:

```typescript
import {
  SourceComponent,
  MetadataApiDeployStatus,
  FileProperties
} from '@salesforce/source-deploy-retrieve'
```

I think `MetadataApiDeployStatus` should be exported at the top level...

Reply:
I see! I'll create a PR for SDR to export `MetadataApiDeployStatus` at the top level. On a similar note, I use `RequestStatus` in `persistentStorageService.test.ts` and it is not currently exported at the top level.  Shall we also export `RequestStatus` at the top level for SDR?